yinsolidated/parser.py

```diff
@@ -154,16 +165,17 @@ def _change_all_whitespace_to_spaces(string):
     return re.sub(r'\s+', ' ', string).strip()
 
 
-class MissingPrefixError(Error):
+class MissingAttributeError(Error):
```

Comment:
I'm not sure anyone is really using this library besides us, but we probably shouldn't remove a public class because it could break existing code. Maybe leave `MissingPrefixError` but have it inherit from your new class?